test/PrintAsObjC/availability.swift

```diff
@@ -45,11 +45,29 @@
 // CHECK-DAG: SWIFT_AVAILABILITY(ios_app_extension,unavailable)
 // CHECK-DAG: SWIFT_AVAILABILITY(tvos_app_extension,unavailable)
 // CHECK-DAG: SWIFT_AVAILABILITY(watchos_app_extension,unavailable)
+// CHECK-DAG: ;
```

Comment:
I think it's okay to leave this out, but if you want to have it you should use `CHECK-SAME` rather than `CHECK-DAG`. The "DAG" suffix means it's reorderable with other `CHECK-DAG` lines immediately before and after this one.

Reply:
Oh I just learn about `CHECK-SAME`, thanks